api/edge_api/identities/tasks.py

```diff
@@ -71,7 +72,7 @@ def call_environment_webhook_for_feature_state_change(
     call_environment_webhooks(environment, data, event_type=event_type)
 
 
-@register_task_handler()
+@register_task_handler(priority=TaskPriority.HIGHEST)
```

Comment:
Does this need to be highest? I think we basically want to reserve highest for those tasks involved in creating the environment document, right?

Reply:
Think this is as import as environment document, right? We want the identity overrides to reflect as soon as possible

Reply:
I'd say it's _slightly_ lower priority that environment document changes.